lops/lop-domain-a72.dts

```diff
@@ -54,6 +54,51 @@
                 lop_8 {
                       compatible = "system-device-tree-v1,lop,code-v1";
                       code = "
+                          assist_dir = os.path.dirname(os.path.realpath(__file__)) + '/assists/'
+                          sys.path.append(assist_dir)
+                          from domain_access import get_domain_nodes, get_mem_nodes, update_mem_node
+                          from baremetalconfig_xlnx import scan_reg_size
+
+                          domain_nodes = get_domain_nodes(tree)
+                          invalid_phandles = []
+                          shared_phandles = []
+                          linux_mem_domain = []
+                          for domain_node in domain_nodes:
+                              linux_domain = []
+                              try:
+                                  if re.search('linux', domain_node['os,type'].value):
+                                      linux_domain = domain_node
+                                      linux_mem_domain = domain_node
+                              except:
+                                  try:
+                                      if domain_node['cpus'].value:
+                                          print('ERROR: os,type property is missing in the domain', domain_node.name)
+                                  except:
+                                      pass
+
+                              if linux_domain:
+                                  try:
+                                      shared_phandles.extend(domain_node['access'].value)
```

Comment:
Note: that for domains that come in via dts versus yaml, it will be value[0], there's a __getitem__ magic method on the property that is designed to hide this detail. So you can try something like:

domain_node['access'][0]

And it will get you the first item if value is json, a list or just a value.

Reply:
Thanks for the review will update logic